internal/config/validate.go

```diff
@@ -106,7 +107,32 @@ func Validate(path, repoPath string) error {
 
 	err = schema.Validate(conf)
 	if err != nil {
-		return fmt.Errorf("error while validating: %w", err)
+		return fmt.Errorf("error while validating against schema: %w", err)
+	}
+
+	err = validateSchemaExtraRules(path)
+	if err != nil {
+		return fmt.Errorf("error while validating against schema rules: %w", err)
+	}
+
+	return nil
+}
+
+func validateSchemaExtraRules(confPath string) error {
```

Comment:
I don't really like this function to be here because config/validate was not bound to a specific apiVersion but for now, this does the trick

Reply:
@Al-Pragliola you are right, let's move it to a per-api implementation.